### ISSUE TITLE
Add filename  to buttons

### ISSUE THREAD

**Issue by elinw** [ropenscilabs/qcoder#142]
Currently when downloading the files with buttons only the file extension is included.

**Comment by elinw** [comment]
I think it should look like this (example is for the units tab:

```
                options = list(paging = FALSE, dom = "Bfrtip",
                                     buttons = list(list(extend='copy'),
                                     list(extend = 'csv', filename = "qc_units"),
                                      list(extend = 'excel', filename = "qc_units"),
                                      list(extend ='pdf', filename = "qc_units"),
                                      list(extend = 'print'))))

```

**Comment by elinw** [comment]
@Yuiken Would you want to do this?

**Comment by Yuiken** [comment]
Sure

**Comment by elinw** [comment]
#147 PR

**Comment by elinw** [comment]
Closed by #152
